src/network/providers/ethereum.js

```diff
@@ -200,7 +200,7 @@ const fetchAndSaveIdentityAbiFromStorage = async () => {
 
         log.debug('Successfully fetched identity contract abi from storage');
 
-        return (abisByContractName['Identity'] = abiAndMetadata);
+        abisByContractName['Identity'] = abiAndMetadata;
```

Comment:
Please don't remove the `return` statement here.

Reply:
Why do we need to return assignment? It makes no sense and against the best practices